### PR TITLE
Add new properties for getting RTPTransceivers.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

**Review by taste1981 — APPROVED** [review]

File: src/sdk/conference/subscription.js [hunk 1]
```diff
@@ -298,6 +299,18 @@ export class Subscription extends EventDispatcher {
       writable: true,
       value: stream,
     });
+    /**
+     * @member {Owt.Base.TransportSettings} transport
+     * @instance
+     * @readonly
+     * @desc Transport settings for the publication.
```
Comment (taste1981) [inline src/sdk/conference/subscription.js:306]: for the subscription. Same for the line below this one.

File: src/sdk/conference/channel.js [hunk 2]
```diff
@@ -876,25 +877,40 @@ export class ConferencePeerConnectionChannel extends EventDispatcher {
...
-      (trackKind) => this._muteOrUnmute(sessionId, false, false, trackKind),
-      (options) => this._applyOptions(sessionId, options));
+      const transportSettings =
+          new TransportSettings(TransportType.WEBRTC, this._id);
+      transportSettings.rtpTransceivers =
+          this._subscribeTransceivers.get(internalId).transceivers;
+      const subscription = new Subscription(
+          sessionId, mediaStream, transportSettings,
+          () => {
+            this._unsubscribe(internalId);
```
Comment (taste1981) [inline src/sdk/conference/channel.js:887]: resolve the subscribe promises here if stopped?

File: src/sdk/conference/subscription.js (hunk 1 above)
Comment (jianjunz) [reply src/sdk/conference/subscription.js]: Fixed.

File: src/sdk/conference/channel.js (hunk 2 above)
Comment (jianjunz) [reply src/sdk/conference/channel.js:887]: This is the implementation of `subscription.stop()` function. Promise is resolved with the subscription here before `stop` function can be called.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**